buildroot-external/package/occu/scripts/createLicenseHtml.py

```
@@ -0,0 +1,140 @@
+    logFilePath = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'createLicenseHtml.log')
+    logging.basicConfig(filename=logFilePath, encoding='utf-8', level=logging.DEBUG)
+    
+    log.info('Starting license HTML creation process...')
+    global commonLicensesDict
+    commonLicensesDict = libCommonLicenses.buildCommonLicensesDict()
+    
+    legalInfoDir = os.path.join(args.build_dir, 'legal-info')
+    log.debug(f'Parsing manifest.csv from {legalInfoDir}...')
+    manifestEntries = parseManfifest( os.path.join(args.build_dir, 'legal-info','manifest.csv') )
+    with open(args.output, 'w') as f:
+        log.debug(f'Writing written offer.')
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)